timesketch/lib/datastores/elastic.py

```diff
@@ -870,6 +884,15 @@ def flush_queued_events(self):
         self.import_events = []
         return return_dict
 
+    def get_field_mapping(self, index, fields):
```

Comment:
Same here: Can you call self.client.indices.get_field_mapping directly instead of using this method?

Reply:
Confirmed and done 👍